infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -43,6 +43,8 @@ export interface BasePromptOptions {
     skip?: ((state: object) => boolean | Promise<boolean>) | boolean;
 }
 
+let useMatterlabsGeth = false;
```

Comment:
This file is already >800 lines long, and I think that allowing to use global variables in it can break the camel's back. Let's not do that.

Reply:
Created task to deprecate this https://linear.app/matterlabs/issue/PLA-681/unintuitive-behavior-of-setup-database-url-and-eth-client-web3-url-env